### PR TITLE
fix(tailwind): delete group class and update readme

### PR DESCRIPTION
**Pull request by TriStarGod**
Not a valid class - [tailwindlabs/tailwindcss#4666](https://github.com/tailwindlabs/tailwindcss/pull/4666)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [18.x]
```
Comment (Romakita) [inline .github/workflows/build.yml:87]: please  rebase your branch on the master ;)
Theses changes are already done !

**Review by Romakita — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (TriStarGod) [reply .github/workflows/build.yml]: Not sure if I've `rebased`. Was using github as my editor and I'm still a newb using `git`.

**Review by Romakita — APPROVED** [review]

**Comment by TriStarGod** [comment]
@Romakita Maybe for tsed-formio but not for [@tsed/tailwind-formio](https://www.npmjs.com/package/@tsed/tailwind-formio)

**Comment by Romakita** [comment]
Arf maybe the publication has failed

**Comment by Romakita** [comment]
Ok got it. I'll republish a new version ;)

**Comment by Romakita** [comment]
Released ;)

**Pull request merged**